src/styles/custom-antd.css

```diff
@@ -15166,7 +15166,7 @@ textarea.ant-input-number {
   padding: 0 50px;
   color: #212121;
   line-height: 64px;
-  background: linear-gradient(to bottom right, #2e3458 0%, #212121 100%);
+  background: #001529;
```

Comment:
It was mentioned in the .less file but all instances of #001529 in the css file were changed to "linear-gradient(to bottom right, #2e3458 0%, #2121 100%)" to get that dark gradient background